### PR TITLE
added bio

### PR DESCRIPTION
**Pull request by bruhad-dave**

### CODE REVIEW

File: content/group/index.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ _raymond at ebi dot ac dot uk_
 
 **Bruhad Dave** is an intern
```
Comment (jhellewell14) [inline content/group/index.md:23]: Are you going to add a photo?

File: content/group/index.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ _raymond at ebi dot ac dot uk_
...
 
+My work in the lab involves writing a [python package](https://github.com/bacpop/pp-netlib) that handles graph data for the population genomics tool, [PopPUNK](https://github.com/bacpop/PopPUNK), and contributing to the analysis of patterns of pneumococcal genomic variation. My past work has included looking at segmental variants in lab-evolved lines of drug-resistant _E. coli_, assembling and annotating data from nosocomial samples, and investigating the extent, rates and predicted functional effects of single-nucleotide variations in the transcriptome of _Plasmodium falciparum_. I am broadly interested in human health & disease research, and I am especially interested in the intersections of pathogen genomics, host-pathogen interactions, and evolutionary biology.
```
Comment (jhellewell14) [inline content/group/index.md:25]: I would stick a comma after the word rates. Other than that, looks good!

File: content/group/index.md (hunk 1 above)
Comment (bruhad-dave) [reply content/group/index.md:23]: i haven't found a suitable one yet, so not for now

File: content/group/index.md (hunk 2 above)
Comment (bruhad-dave) [reply content/group/index.md]: made the change, thanks!

**Review by jhellewell14 — APPROVED** [review]

**Pull request merged**